app/src/report/reportActions.js

```diff
@@ -103,9 +103,13 @@ export function sendReport() {
       from: state.filters.timelineInnerExtent[0].toISOString(),
       to: state.filters.timelineInnerExtent[1].toISOString()
     };
-    const currentFlags = state.filters.flags.map(flag => flag.flag)
-      .filter(flag => flag !== undefined)
-      .map(flag => FLAGS[flag]);
+    const currentFlags = state.filterGroups.filterGroups.filter(filter =>
+      filter.filterValues !== undefined &&
+      filter.filterValues.category !== undefined &&
```

Comment:
too many comparisons, feels evil. what is this supposed to do?

Reply:
Just map the countries from the filters for the reports. You need to select only the ones that contain a filter values and filtered country. It may be documented or abstracted to other function

Reply:
Ariadna likes the solution but only if we update it when changed